### PR TITLE
[CLOUD-940] Expose the MAVEN_MIRROR_URL variable for JDV

### PR DESCRIPTION
**Pull request by osnipso**
If set, it will be used by s2i while building sources.

### CODE REVIEW

**Comment by osnipso** [comment]
@jwendell did for this one, too - displayName now appears first.

File: datavirt/datavirt63-extensions-support-s2i.json [hunk 1]
```diff
@@ -494,6 +501,10 @@
                             {
                                 "name": "VDB_DIRS",
                                 "value": "${VDB_DIRS}"
+							},
+							{
```
Comment (luksa) [inline datavirt/datavirt63-extensions-support-s2i.json]: Indentation needs to be fixed here.

File: datavirt/datavirt63-basic-s2i.json [hunk 2]
```diff
@@ -140,6 +140,13 @@
             "name": "VDB_DIRS",
             "value": "",
             "required": false
+        },
+        {
+            "displayName": "Maven mirror URL",
+            "description": "Maven mirror to use for s2i builds",
```
Comment (luksa) [inline datavirt/datavirt63-basic-s2i.json]: Maybe change s2i to uppercase.

File: datavirt/datavirt63-extensions-support-s2i.json (hunk 1 above)
Comment (osnipso) [inline datavirt/datavirt63-extensions-support-s2i.json]: Good catch! Done.

**Comment by osnipso** [comment]
Done.

**Comment by osnipso** [comment]
Closing, replaced by [#253](https://github.com/jboss-openshift/application-templates/pull/253)

**Pull request closed without merging**